dbv2/reader.go

```diff
@@ -0,0 +1,189 @@
+package dbv2
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var (
+	readerBufferSize = validLength * (bufferSize / 10)
+	// readerOffsetBytes represents the number of bytes that occupy the meta-data section of the table. This can be discarded
+	// during parse operation as this is only to know the type of the table.
+	readerOffsetBytes = len([]byte(`maxValidLength: 100 chars
```

Comment:
What does the table metadata comprise? The hash value of routes?

Reply:
I will write a design doc for that. Mentioning it here will be very confusing to understand.